pgx-tests/src/tests/fcinfo_tests.rs

```diff
@@ -128,46 +126,13 @@ fn fcinfo_not_named_no_arg(fcinfo: pg_sys::FunctionCallInfo) -> i32 {
     todo!()
 }
 
-#[derive(PostgresType, Serialize, Deserialize, Debug, PartialEq)]
-#[inoutfuncs]
-pub struct NullStrict {}
```

Comment:
I think this was deleted for reasons that it no longer needs to be.  I'll be bringing it back...